test/jdk/javax/net/ssl/SSLException/CheckSSLHandshakeException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2022 THL A29 Limited, a Tencent company. All rights reserved.
```

Comment:
I am unsure about the copyrights on these files.  They are probably ok, but you should get approval by someone more familiar.

Reply:
The location of the copyright is fine.  No need to enforce Oracle date formatting.